deid/dicom/utils.py

```diff
@@ -64,21 +64,23 @@ def get_files(contenders, check=True, pattern=None, force=False, tempdir=None):
             if dextension == ".zip":
                 with zipfile.ZipFile(dicom_file, "r") as compressedFile:
                     compressedFile.extractall(tempdir)
-                    dicom_file = next(
-                        os.path.join(tempdir, f)
-                        for f in os.listdir(tempdir)
-                        if os.path.isfile(os.path.join(tempdir, f))
-                    )
-
-            if dicom_file is not None:
```

Comment:
Looks like you forgot to add back the check here.

Reply:
Not at all. It's useless. See commments in commit.

Reply:
@BrianKolowitz it looks like you added this check - can you confirm the change looks okay to you?

Reply:
Maybe it wasn't or appeared not to be useless before, now it is useless because of the additional try/except.

Reply:
`dicom_file` cannot be empty because we `continue` in case of `StopIteration`.

Reply:
Understood, and it LGTM - I'm just hoping to get the blessing of the original contributor before changing the code he wrote.

Reply:
Sure, I'd like to have it too.

Reply:
@wetzelj it looks like @BrianKolowitz is not able to respond (I hope he's okay!) and I'd like to get one more review on this before merge. It's fairly small changes - if you have time in the next few weeks could you take a look?